core/src/main/java/com/datastrato/gravitino/GravitinoEnv.java

```diff
@@ -139,7 +139,8 @@ public void initialize(Config config) {
     LOG.info("Initializing Gravitino Environment...");
 
     this.config = config;
-    this.metricsSystem = new MetricsSystem();
+    int slidingWindowSeconds = config.get(Configs.METRICS_TIME_SLIDING_WINDOW_SECONDS).intValue();
+    this.metricsSystem = new MetricsSystem(slidingWindowSeconds);
```

Comment:
Why don't you pass in a `Config` object as a class parameter and get the specific configuration in `MetricsSystem`, it is a little strange to get the configuration here in `GravitinoEnv`.